YahooStockData/StockAnalysis/historical_bars.py

```diff
@@ -0,0 +1,42 @@
+__author__ = 'byron'
+
+import datetime
+
+class TickerData():
+    def __init__(self, ticker, time_frame, exchange):
+        self.ticker = ticker
+        self.time_frame = time_frame
+        self.exchange = exchange
+        self.candlesticks = []
+        self.file_contents = self.get_ticker_data()
+
+
+    def get_ticker_data(self):
+        __folder = '/home/byron/Stock-Data/{0}'.format(self.exchange)
+        __file_name = '{0}/{1}-{2}.data'.format(__folder, self.ticker, self.time_frame)
+
+        with open(__file_name, mode='r') as data_file:
```

Comment:
We can just list it as miscellaneous since it does not fit into the other categories.